quickstart/speech-translation/csharp-dotnetcore/README.md

```diff
@@ -0,0 +1,64 @@
+# Quickstart: Translate speech in C# for .NET Core (Windows or Linux)
+
+This sample demonstrates how to translate speech with C# under .NET Core 2.0 (Windows or Linux) using the Speech SDK.
+See the [accompanying article](https://docs.microsoft.com/azure/cognitive-services/speech-service/quickstart-translate-speech-dotnetcore-windows) on the SDK documentation page which describes how to build this sample from scratch in Visual Studio 2017.
+
+> **Note:**
+> on Linux, we currently only support the Ubuntu 16.04 x64 distribution.
```

Comment:
add Ubuntu 18.04

Reply:
also below

---
In reply to: [252246004](https://github.com/Azure-Samples/cognitive-services-speech-sdk/pull/142#discussion_r252246004) [](ancestors = 252246004)